floodviz/static/js/peakflow.js

```diff
@@ -53,17 +57,29 @@ document.addEventListener("DOMContentLoaded", function(event) {
 			.attr("x", 0 - (height / 2))
 			.attr("y", 0 - (margin.left / 2))
 			.text("Discharge (cfps)");
-		
+
+		// Save last data point as lollipop
 		var lolli_data = data[data.length - 1];
+		// remove last data point for creating bars
 		data = data.slice(0, data.length - 1);
 
+
 		// Normal Bar value creation
 		svg.selectAll("bar").data(data).enter().append("rect")
 			.attr('class', 'bar')
 			.attr("x", function(d) {return x(d.label); })
 			.attr("y", function(d) {return y(d.value); })
 			.attr("width", x.bandwidth())
-			.attr("height", function(d) {return height - y(d.value); });
+			.attr("height", function(d) {return height - y(d.value); })
+			.on("mousemove", function(d) {
+				tooltip.transition().duration(500).style("opacity", .9);
+				tooltip.style("display", "inline-block")
+                .style("left", (d3.event.pageX) + 10 + "px")		
```

Comment:
Not sure why the spacing is like this on github. But the actual file spacing is correct.